modules/oadp-creating-restore-cr.adoc

```diff
@@ -36,10 +36,11 @@ spec:
   - backups.velero.io
   - restores.velero.io
   - resticrepositories.velero.io
-  restorePVs: true
+  restorePVs: true <3>
 ----
 <1> Name of the `Backup` CR.
 <2> Optional. Specify an array of resources to include in the restore process. Resources might be shortcuts (for example, 'po' for 'pods') or fully-qualified. If unspecified, all resources are included.
+<3> restorePVs can be set to `false` in order to turn off backup of `PersistentVolumes`.
```

Comment:
```suggestion
<3> restorePVs can be set to `false` in order to turn off restore of `PersistentVolumes`.
```
/cc @shubham-pampattiwar

Reply:
Nope, it only specifies whether to restore the PVs from snapshots

Reply:
@shubham-pampattiwar @kaovilai 
Please explain what I should write for the description.

Reply:
Maybe re-frame to:  `restorePVs` can be set to `false` in order to turn off restore of `PersistentVolumes` from `VolumeSnapshots`.